nuttx-configs/s2740vc-v1/Kconfig

```diff
@@ -3,6 +3,6 @@
 # see misc/tools/kconfig-language.txt.
 #
 
-if CONFIG_ARCH_BOARD_S2740VC
+if CONFIG_ARCH_BOARD_S2740VC_V1
```

Comment:
These are all nuttx formatting and patterns. That is way the names were on the path configs/ it is from nuttx/configs - as it the copyright on most of the files. The root name is the board/family let's not change those. If all the file are called init.c then the tabs in the editor become [init.c][init.c][init.c]  and that is not good. It is like when you have 10 CMakeLists.txt file open. 

Let's not do any more formatting at this point either.

formmating either